### PR TITLE
Feature/google sso

### PR DESCRIPTION
**Pull request by MaxGudinovich**
Created google sso.
Refactoring spaces.

### CODE REVIEW

File: News/Configurations/CommentConfiguration.cs
Comment (IKonrad98) [inline News/Configurations/CommentConfiguration.cs:1]: Why is part of the code missing here?
  
```c#   
builder.ToTable("Comments");
....
builder.HasOne(e => e.User)
            .WithMany(e => e.Comments)
            .HasForeignKey(e => e.UserId);
```

Comment (MaxGudinovich) [reply News/Configurations/CommentConfiguration.cs:1]: I dont know, Visual bag mb. I have that and double checked at code.
 builder.ToTable("Comments");
        builder.HasKey(e => e.Id);
        builder.HasOne(e => e.User)
            .WithMany(e => e.Comments)
            .HasForeignKey(e => e.UserId);

File: News/Entities/ArticleEntity.cs
Comment (IKonrad98) [inline News/Entities/ArticleEntity.cs:1]: Why did you add empty spaces between the lines?

File: News/Migrations/20241013080115_NullablePassworId.cs
Comment (IKonrad98) [inline News/Migrations/20241013080115_NullablePassworId.cs:1]: Migrations need to be added to the .gitignore file

File: News/Controllers/CommentsController.cs [hunk 1]
```diff
@@ -16,45 +16,32 @@ public CommentsController(ICommentService service)
         _service = service;
     }
 
-    [HttpPost]
-    public async Task<IActionResult> CreateAsync(
-        [FromBody] CommentCreateModel comment,
-        CancellationToken cancellationToken
-        )
+    [HttpGet("{id}")]
+    public async Task<IActionResult> GetCommentById([FromRoute] Guid id, CancellationToken cancellationToken)
```
Comment (IKonrad98) [inline News/Controllers/CommentsController.cs]: Why is the old name here? 
Must be GetByIdAsync

File: News/Controllers/CommentsController.cs
Comment (IKonrad98) [inline News/Controllers/CommentsController.cs:1]: Old method names

File: News/Controllers/GoogleAuthController.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{
+    private readonly IGoogleAuthService _googleAuthService;
+
+    public GoogleAuthController(IGoogleAuthService googleAuthService)
+    {
+        _googleAuthService = googleAuthService;
+    }
+
+    [HttpPost("login")]
+    public async Task<IActionResult> Login([FromBody] string googleAccess, CancellationToken cancellationToken)
```
Comment (IKonrad98) [inline News/Controllers/GoogleAuthController.cs]: LoginAsync

File: News/DataAccess/Repo/PasswordRepo.cs [hunk 3]
```diff
@@ -28,7 +28,7 @@ CancellationToken cancellationToken
     }
 
     public async Task<PasswordEntity> GetByIdAsync(
-        Guid id,
+        Guid? id,
```
Comment (AvaNice) [inline News/DataAccess/Repo/PasswordRepo.cs:31]: do we really need a nullable guid here?

File: News/Migrations/20241013080115_NullablePassworId.cs [hunk 4]
```diff
@@ -0,0 +1,80 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+
+#nullable disable
+
+namespace News.Migrations;
+
+/// <inheritdoc />
+public partial class NullablePassworId : Migration
```
Comment (AvaNice) [inline News/Migrations/20241013080115_NullablePassworId.cs:9]: remove migrations

**Review by AvaNice — APPROVED** [review]

**Review by IKonrad98 — APPROVED** [review]
